src/runtime/virtcontainers/kata_agent.go

```diff
@@ -1662,6 +1669,60 @@ func (k *kataAgent) createBlkStorageObject(c *Container, m Mount) (*grpc.Storage
 	return vol, err
 }
 
+// handleSMBMounts will create a unique destination mountpoint in the guest for each volume in the
+// given container and will update the OCI spec to utilize this mount point as the new source for the
+// container volume. The container mount structure is updated to store the guest destination mountpoint.
+func (k *kataAgent) handleSMBMounts(c *Container, spec *specs.Spec) ([]*grpc.Storage, error) {
+	var volumeStorages []*grpc.Storage
+	for i, m := range c.mounts {
+		// Handle only cifs type of mounts
+		if m.Type != "cifs" {
```

Comment:
This comes from the CSI driver, right?

Reply:
Yes, this comes from the `mountInfo.json` which is written to by the CSI driver and parsed by kata shim.